DualView/src/main/java/com/microsoft/device/display/samples/dualview/view/RestaurantAdapter.kt

```diff
@@ -82,17 +99,14 @@ class RestaurantAdapter(
          * Changes the text color of the view to black if the cell is selected, otherwise to gray
```

Comment:
I don't think this comment is valid anymore, we should either update it or remove it

Reply:
Updated the comment.